source/Calamari.Tests/Fixtures/Integration/FileSystem/CalamariPhysicalFileSystemFixture.cs

```diff
@@ -132,14 +130,29 @@ string CreateFile(params string[] relativePath)
         [TestCase(@"**/*.txt", "f1.txt", 2)]
         [TestCase(@"**/*.txt", "r.txt", 2)]
         [TestCase(@"*.txt", "r.txt")]
-        [TestCase(@"**/*.config", "root.config", 5)]
+        [TestCase(@"**/*.config", "root.config", 6)]
         [TestCase(@"*.config", "root.config")]
         [TestCase(@"Config/*.config", "c.config")]
-        [TestCase(@"Config/Feature1/*.config", "f1-a.config", 2)]
-        [TestCase(@"Config/Feature1/*.config", "f1-b.config", 2)]
+        [TestCase(@"Config/Feature1/*.config", "f1-a.config", 3)]
+        [TestCase(@"Config/Feature1/*.config", "f1-b.config", 3)]
+        [TestCase(@"Config/Feature1/*.config", "f1-c.config", 3)]
         [TestCase(@"Config/Feature2/*.config", "f2.config")]
-        public void EnumerateFilesWithGlob(string pattern, string expectedFileMatchName, int expectedQty = 1)
+        [TestCase(@"Config/Feature1/*-{a,b}.config", "f1-a.config", 2, 2)]
+        [TestCase(@"Config/Feature1/*-{a,b}.config", "f1-b.config", 2, 2)]
```

Comment:
I wrote these tests after I started changing the `Glob` class. If I had put them in first, I would have seen that the current implementation handles this specific case (having the `*` before the `{}` group).